cfep-06.md

```diff
@@ -0,0 +1,147 @@
+
+Our compromise on the call was that we should have an automated system for *labeling* PRs and let
+humans ultimately decide *when* to close the PR. The conversation was sufficiently complete that
+we thought a CFEP was appropriate so that all of @conda-forge/core and
+@conda-forge/staged-recipes had an opportunity to weigh in before we implement anything. The
+thinking is that by defining which **labels** we should have and what triggers a **transition**
+between labels we can clearly signify (1) whether the **Reviewers** or the **Authors** are
+supposed to take action and (2) whether or not we have given the **Author** sufficient time to
+address the review that we can close the PR as it has become stale and inactive.
+
+The desired outcome of this CFEP is an explicit definition of what a "stale" PR is such that we
+can all feel okay with closing someone elses hard work. We will produce this explicit definition
+of what "stale" means by codifying the code-review process into **labels** and **transitions**
+between these labels. After this has been defined and agreed up on we can implement a bot based
+on this CFEP.
+
+## Terminology
+
+* **Author**: The person who opens the pull request against staged-recipes
+* **Reviewer**: Anyone with commit rights into conda-forge/staged-recipes. As of now (July 2018), these
+  are the core and staged-recipes teams
+* **labels**: Nodes in the state-machine. Not all nodes will have a label (merged and closed, for
+  example)
+* **transition**: The specific action that triggers moving between nodes in the state machine
+
+## Specification
+
+```
+(Open new PR) > 0 > [Awaiting Review] < < <
+                            ˅             ^
+                            1             ^
+                            ˅             ^
+            > > > [Awaiting Author] > 2 > ^
+            ^               ˅             ^
+            ^               3             ^
+            ^               ˅             ^
+            ^ < < 4 < < [Stale] > > 5 > > ^
+            ^               ˅             ^
+            ^               6             ^
+            ^               ˅             ^
+            ^ < 7 < [Can be closed] > 8 > ^
+                           ˅
+                           9
+                           ˅
+                        [Closed]
+```
+
+### States and Transitions
+
+We have six proposed states. All but the "closed" and "merged" states should have an associated
+github label with them. Merged can be reached from any state other than "Closed" so it is not
+worth putting on to this state machine diagram.
+
+* Awaiting Review
+* Awaiting Author
+* Stale
+* Can be closed
+* Closed
+* Merged
+
+We have 10 proposed transitions. The syntax here is `+` or `-` followed by a label. This indicates
+that a specific label is added or removed.
+
+0. **Author** submits pull-request to staged-recipes
+    * +"Awaiting Review"
+1. A **Reviewer** reviews the recipe and requests changes.
```

Comment:
This seems to imply (and probably *should* imply) that reviews should either "Request changes" (in the github PR review sense) or "Approve".  If any feedback should trigger "Awaiting Author" then maybe change the wording here to "reviews the recipe".

As a corollary, should reviews be instructed to *not* use the "Comment" option when reviewing?